### ISSUE TITLE
Filter is letting lots of empty lines pass through

### ISSUE THREAD

**Issue by janwilmans** [madskristensen/OutputWindowFilter#7]
**Describe the bug**
If I enter a filter, I do not just get lines with the filtered word on them, but also lots of empty lines.

**To Reproduce**
see #6

**Additional context**

This is actually causing confusion that caused me to report #6

**Comment by Black-Platypus** [comment]
This is \*probably\* because this extension is not actually removing lines but just applies a text filter with a font size of 0.00001 and a foreground color of transparent. 
https://github.com/madskristensen/OutputWindowFilter/blob/a3b20f20399995321b777bb3aebd9889222e9dfb/src/ClassificationTypeDefinitions.cs#L23-L28
I was pretty surprised to see this when I looked into it, but I currently don't have a drop-in alternative.
The line elements must have some other property to them, like padding/margin that can't be affected with a text filter, "from the inside", as it were, so we're left with about 0.5% height of the hidden lines still taking up space.
This also means when you copy a section with hidden lines, you'll get the full text with things you thought you filtered out.
This is part of a larger problem that lies at the very base of the approach

**Comment by philippesanchescotch** [comment]
Yes I find this issue annoying as well, I understand why the font/resize approach was used (I consider it mostly a hack, but in the end it there might be no other approaches to this atm).

Maybe creating a different output window for your extension which read it's input from the main output window and apply a filter on it's content, while removing the empty lines ?
